src/main/java/frc/robot/RobotMap.java

```diff
@@ -36,5 +39,11 @@
         int TRANSFER_ENTRANCE_LIGHT_SENSOR = 2;
 
         int INTAKE_TO_TRANSFER_LIMIT = 3;
+
+        // @TODO: change to real ports
+        int CLIMBER_PLACER_LEFT_LIMIT_FRONT = -1;
```

Comment:
check with mechatronics if all of those limits will be in the robot, maybe they forgot them

Reply:
ok, there is no limit switches for the placers for this district.